docs/modules/migrate/pages/dmt-command-reference.adoc

```diff
@@ -0,0 +1,90 @@
+= dmt
+
+dmt commands are a group of migration operations.
```

Comment:
The DMT commands allow you to manage your migration.

The DMT commands have the following format:

Reply:
https://github.com/hazelcast/hz-docs/pull/890/commits/7e48b6af17c9f43c94a9570a9da520a87ec82cf1